recipes/openblas/all/conanfile.py

```diff
@@ -19,134 +23,221 @@ class OpenblasConan(ConanFile):
         "fPIC": [True, False],
         "build_lapack": [True, False],
         "use_thread": [True, False],
+        "use_openmp": [True, False],
         "dynamic_arch": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "build_lapack": False,
         "use_thread": True,
+        "use_openmp": False,
         "dynamic_arch": False,
     }
-    generators = "cmake"
+    package_type = "library"
     short_paths = True
 
+    def _fortran_runtime(self, fortran_id):
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            if fortran_id == "GNU":
+                if self.settings.compiler == "gcc":
+                    if Version(self.settings.compiler.version).major in ["5", "6"]:
+                        return "gfortran"  # Runtime version gfortran3
+                    if Version(self.settings.compiler.version).major == "7":
+                        return "gfortran"  # Runtime version gfortran4
+                    if Version(self.settings.compiler.version).major > "7":
+                        return "gfortran"  # Runtime version gfortran5
```

Comment:
This is some verbose logic to ultimately return `"gfortran"` - is there a reason it needs to be segmented like this? Would something like the following be sufficient here?
```suggestion
                    if Version(self.settings.compiler.version).major > "5":
                        return "gfortran"
```

Reply:
No reason, other than compiler versions require different runtime versions of gfortran. I applied your suggested change.